From ddbfbdf09e3ee61b3901e0c9398e448a2bb1f471 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Tue, 2 Feb 2016 22:04:04 +0200
Subject: [PATCH 0778/1240] dts: 7040rz: enable comphy for Armada-7040RZ DOP
 board

Lane 0: UNCONNECTED
Lane 1: USB3-HOST0
Lane 2: UNCONNECTED
Lane 3: SATA1
Lane 4: USB3-HOST1
Lane 5: PEX2x1

Change-Id: I573271fa7e62fab6e86ad548acd4fe251f3e36a9
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27164
Reviewed-by: Shadi Ammouri <shadi@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
---
 arch/arm/dts/armada-7040-rz-db_dop.dts | 33 ++++++++++++++++++++++++++++++++-
 1 file changed, 32 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/armada-7040-rz-db_dop.dts b/arch/arm/dts/armada-7040-rz-db_dop.dts
index 895e7d6..506cd21 100644
--- a/arch/arm/dts/armada-7040-rz-db_dop.dts
+++ b/arch/arm/dts/armada-7040-rz-db_dop.dts
@@ -96,7 +96,38 @@
 					     0xE 0xE 0xE 0>;
 			};
 			comphy {
-				status = "disabled";
+				/* Serdes Configuration:
+					Lane 0: UNCONNECTED
+					Lane 1: USB3-HOST0
+					Lane 2: UNCONNECTED
+					Lane 3: SATA1
+					Lane 4: USB3-HOST1
+					Lane 5: PEX2x1
+				*/
+				phy0 {
+					phy-type = <PHY_TYPE_UNCONNECTED>;
+					phy-speed = <PHY_SPEED_1_25G>;
+				};
+				phy1 {
+					phy-type = <PHY_TYPE_USB3_HOST0>;
+					phy-speed = <PHY_SPEED_5G>;
+				};
+				phy2 {
+					phy-type = <PHY_TYPE_UNCONNECTED>;
+					phy-speed = <PHY_SPEED_1_25G>;
+				};
+				phy3 {
+					phy-type = <PHY_TYPE_SATA1>;
+					phy-speed = <PHY_SPEED_5G>;
+				};
+				phy4 {
+					phy-type = <PHY_TYPE_PEX0>;
+					phy-speed = <PHY_SPEED_5G>;
+				};
+				phy5 {
+					phy-type = <PHY_TYPE_PEX1>;
+					phy-speed = <PHY_SPEED_5G>;
+				};
 			};
 
 		};
-- 
1.9.1

